script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -642,26 +668,26 @@ It's a matter of %{i}how%{/i} I can kill him.
 [sha:55223c9ede27228eb3cf55feaa0065fd89abc45f]{
 -- Page 641, Offset 10381.
 -- 覚悟を決めろ。
-I must steel my resolve.
+It's time to commit.
 }
 [sha:e106adb3410aa834dbd2f5ff8b52f0b89f98f8f6]{
 -- Page 641, Offset 10382. Mods: @k, @e.
 -- 接近する手段があるとしたら、それは、
-If there is any way to get close, it's by...
+There's only one way I can get close, and it's―――
 }
 [sha:821abfcb90bf6332464cf07fd40338be40513e72]{
 -- Page 643, Offset 42621. Choice.
 -- ……投擲だ。
- ...Throwing.
+ Throwing.
 }
 [sha:0af47ce94447969d3bd6fa2daf1dd4b3fa54723e]{
 -- Page 643, Offset 42622. Choice.
 -- ……滑走だ。
 // 滑走, gets updated to 落下 inside the file
- ...Gliding.
+ Gliding.
```

Comment:
Still think sliding is correct, tbh.

Reply:
None of what he really does is _sliding_ though, right? Gliding here to me is as in planes gliding [through the air], rather than gliding [on ice], which feels more correct to me based on what he does next / connects better to 落下.

Reply:
It's supposed to be related to 滑走路, i.e, he uses the rock formation as an impromptu runway, and that doesn't really feel like gliding. Think of it how you can like 'slide down the side of a mountain'. You're falling, sure, but since you're not in freefall (still in contact with the mountain) you're not falling, but sliding.

Reply:
You don't slide down a runway though. You coast, glide, taxi, etc.

Reply:
SLIDE SLIDE SLIDE SLIDE